composer.json

```diff
@@ -60,11 +60,22 @@
                     "overwrite": false,
                     "path": "docroot/core/assets/scaffold/files/default.services.yml"
                 },
+                "[web-root]/sites/default/services.yml": {
+                    "mode": "replace",
+                    "overwrite": false,
+                    "path": "docroot/core/assets/scaffold/files/default.services.yml"
+                },
                 "[web-root]/sites/default/default.settings.php": {
                     "mode": "replace",
                     "overwrite": false,
                     "path": "docroot/core/assets/scaffold/files/default.settings.php"
+                },
+                "[web-root]/sites/default/settings.php": {
+                    "mode": "replace",
+                    "overwrite": false,
+                    "path": "docroot/core/assets/scaffold/files/default.settings.php"
```

Comment:
Unfortunately this doesn't include the required items for integrating with Acquia Cloud. I guess it can make that easier. Will it fix the ACMS patching?

Reply:
i have no clue. theoretically if the settings file is there, acquia cloud should automatically inject what is necessary to run the database settings. i think.

Reply:
It seems like I had to copy what ACMS tries to patch for it to work. 

I think it would make sense if ACMS provided it as `settings.acquia.php` which the default `settings.php` check if it existed and included it. Separate concerns. Make it maintainable